x-pack/plugins/infra/public/plugin.ts

```diff
@@ -210,8 +224,9 @@ export class Plugin implements InfraClientPluginClass {
       order: 8200,
       appRoute: '/app/metrics',
       category: DEFAULT_APP_CATEGORIES.observability,
+      updater$: this.appUpdater$,
       // !! Need to be kept in sync with the routes in x-pack/plugins/infra/public/pages/metrics/index.tsx
```

Comment:
This is technically a little out of scope, but I was reading a bit more and wondering if this comment should get moved above the `infraDeepLinks` definition. I think what it's getting at is that we need make sure the deep links list doesn't deviate from the route definitions.

The deep links are defined starting back on line 188 so the comment might make more sense there.